lnbits/static/js/wallet.js

```diff
@@ -4,6 +4,7 @@ window.WalletPageLogic = {
     return {
       origin: window.location.origin,
       baseUrl: `${window.location.protocol}//${window.location.host}/`,
+      websocketUrl: `${'http:' ? 'ws://' : 'wss://'}${window.location.host}/api/v1/ws`,
```

Comment:
websocketUrl is available as global via base.html 
https://github.com/lnbits/lnbits/blob/dev/lnbits/templates/base.html#L439

Reply:
Doesnt work, which is why i added to wallet